demoapp/demoapp/views.py

```diff
@@ -0,0 +1,22 @@
+from django.views.generic import RedirectView
+
+from demoapp.models import SubscriberEmail
+from django.shortcuts import redirect
+
+
+class NewsletterUnsubscribeView(RedirectView):
+    def get_redirect_url(self, *args, **kwargs):
+        return redirect('/')
```

Comment:
This is supposed to return the redirect *URL*, not the HTTP response or Django view.